landing-page/content/common/vendors.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Vendors"
+url: vendors
+disableSidebar: true
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+## Vendors shipping Apache Iceberg
+
+This page contains some of the vendors who are shipping Apache Iceberg in their products
+
+
+### [Cloudera](http://clouera.com)
+
+Cloudera Data Platform integrates Apache Iceberg to the following components:
```

Comment:
I like the content here! Later on we can put this in a shortcode that will add some custom styling and also make this more like a form to keep every new addition consistent. That way it's easier to just fill out name, link, description, feature-list, code-snippet, etc with character limits built into the styling.